schema/rdl_schema_0.1.json

```diff
@@ -1025,6 +1025,37 @@
 				"Judgement",
 				"Hybrid"
 			]
+		},
+		"Temporal": {
+			"title": "Temporal coverage",
+			"type": "object",
+			"description": "The temporal period that the dataset covers.",
+			"properties": {
+				"start": {
+					"title": "Start date",
+					"type": "string",
+					"description": "The date on which the period begins. This can refer to a year (YYYY), a month (YYYY-MM) or a day (YYYY-MM-DD) as appropriate for the dataset.",
+					"format": "date-time"
+				},
+				"end": {
+					"title": "End date",
+					"type": "string",
+					"description": "The date on which the period ends. This can refer to a year (YYYY), a month (YYYY-MM) or a day (YYYY-MM-DD) as appropriate for the dataset.",
+					"format": "date-time"
+				},
+				"duration": {
+					"title": "Duration",
+					"type": "string",
+					"description": "The total number of time periods covered using ISO 8601 duration format, e.g. 50 years is given as 'P50Y'.",
+					"format": "date-time"
+				},
+				"year": {
+					"title": "Reference year",
+					"type": "integer",
+					"description": "A general reference year to which the modelled scenario(s) refers (e.g. '2050')"
```

Comment:
I think that we need to clarify the semantics of this field, i.e. what is a 'general reference year'?

It's not clear to me what the purpose of this field is when the year(s) covered by the period can be expressed using `start` and `end`.

Reply:
see https://github.com/GFDRR/rdl-standard/issues/62#issuecomment-1612957557 we're removing this field